doc/nrf/ug_thread_power_consumption.rst

```diff
@@ -0,0 +1,102 @@
+.. _thread_power_consumption:
+
+OpenThread power consumption
+############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This page provides information about the amount of power used by Sleep End Devices (SEDs) and Synchronized Sleep End Devices (SSEDs) using OpenThread stack.
```

Comment:
```suggestion
This page provides information about the amount of power used by Sleepy End Devices (SEDs) and Synchronized Sleepy End Devices (SSEDs) that use the OpenThread stack.
```
End change is so that the sentence can't be read "provides information [...] using the OpenThread stack".